_maps/map_files/tramstation/tramstation.dmm

```diff
@@ -18670,11 +18660,11 @@
 /turf/open/floor/iron,
 /area/maintenance/disposal/incinerator)
 "ekC" = (
-/obj/effect/turf_decal/sand/plating,
-/obj/structure/disposalpipe/segment{
-	dir = 4
+/obj/machinery/computer/pod/old/mass_driver_controller/trash{
+	pixel_x = -5;
+	pixel_y = -5
 	},
-/turf/open/floor/plating/airless,
+/turf/closed/wall,
```

Comment:
this is wrong, your mass driver controller is directly on the wall, this isnt how wallmounts work, they are to be placed on the floor in front of the wall, and then pixel shifted to look like they are on the wall, make sure your button is not directly on the wall as well

also, i notice this wall replaced an airless plating.... can you make sure no plating inside the actual disposals room is airless? they should be normal plating

Reply:
I've skimmed through the instances on the map for /plating/airless after checking disposals and it just seems like a one off error, I don't see any more on the station

Reply:
ps you have eyes like an eagle, how tf did you spot that not being on the wall from code??

Reply:
> ps you have eyes like an eagle, how tf did you spot that not being on the wall from code??

do you see those three letters ? ``"xBh={`` for example? those denote a tile. both a wall and the launch controller were listed on the same tile. and the launch controller had -5x, -5y which perfectly coincides with being in the bottom left of a tile